README.rst

```diff
@@ -272,6 +272,18 @@ Configuration variables can be set via Env variables:
 - ``WATCHER_SCALYR_JOURNALD_EXTRA_FIELDS``: Add extra Systemd Journald fields. Should be a JSON string. Example: '{"_COMM": "command"}'
 - ``WATCHER_SCALYR_JOURNALD_PATH``: Journald logs path mounted from the host. (Default: ``/var/log/journald``)
 
+Scalyr custom parser
+....................
+
+In some cases you might need to assign a **custom** Scalyr parser for specific container. This can be achieved via Pod annotations. The following example shows an annotation value that instructs kubernetes-log-watcher to set custom parser ``json-java-parser`` for container ``app-1``.
+
+.. code-block:: yaml
+
+  annotations:
+    kubernetes-log-watcher/scalyr-parser: '[{"container": "app-1", "parser": "json-java-parser"}]'
```

Comment:
I couldn't find a reference that recommends ``kubernetes.io`` or dns like namespacing for annotations (although I remember there was one at some point, may be I can't find it now). In the docs it is described similar to labels. If generic keys are accepted now, I believe it would be better than using ``kubernetes.io`` or other specific domains (i.e. ``<project-name>/<component>``)